site/content/en/docs/v0.1/quickstart.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Quickstart
+weight: 0
+description: Get etcd with etcd-operator up and running in less than 5 minutes!
+---
+
+Follow these instructions to install, run, and test etcd with etcd-operator in a Kubernetes cluster.
+
+Pre-requisites:
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Kubernetes cluster and `kubectl` configured to use it
+  - If you don't have a Kubernetes cluster, you can use [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create a local one
+
+1. Install etcd-operator:
+    ```bash
+    kustomize build 'https://github.com/aenix-io/etcd-operator//config/default?ref=main' | kubectl apply -n etcd-operator -f -
```

Comment:
It might be better to use built-in kustomize client in kubectl intead of external one

```suggestion
    kubectl apply -n etcd-operator -k 'https://github.com/aenix-io/etcd-operator//config/default?ref=main' 
```

However I recieve an error:

```
error: trouble configuring builtin PatchTransformer with config: `
path: webhookcainjection_patch.yaml
`: unable to parse SM or JSON patch from [# This patch add annotation to admission webhook config and
# CERTIFICATE_NAMESPACE and CERTIFICATE_NAME will be substituted by kustomize
apiVersion: admissionregistration.k8s.io/v1
kind: MutatingWebhookConfiguration
metadata:
  labels:
    app.kubernetes.io/name: mutatingwebhookconfiguration
    app.kubernetes.io/instance: mutating-webhook-configuration
    app.kubernetes.io/component: webhook
    app.kubernetes.io/created-by: etcd-operator
    app.kubernetes.io/part-of: etcd-operator
    app.kubernetes.io/managed-by: kustomize
  name: mutating-webhook-configuration
  annotations:
    cert-manager.io/inject-ca-from: CERTIFICATE_NAMESPACE/CERTIFICATE_NAME
---
apiVersion: admissionregistration.k8s.io/v1
kind: ValidatingWebhookConfiguration
metadata:
  labels:
    app.kubernetes.io/name: validatingwebhookconfiguration
    app.kubernetes.io/instance: validating-webhook-configuration
    app.kubernetes.io/component: webhook
    app.kubernetes.io/created-by: etcd-operator
    app.kubernetes.io/part-of: etcd-operator
    app.kubernetes.io/managed-by: kustomize
  name: validating-webhook-configuration
  annotations:
    cert-manager.io/inject-ca-from: CERTIFICATE_NAMESPACE/CERTIFICATE_NAME
]
```

Sorry I'm not very familiar with kustomize :-)